### DISCUSSION TITLE
Calling peaks in scATAC-seq with Tile Matrix (not using MACS2)

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#581] [Questions / Documentation]
Hi @jgranja24 and @rcorces ,

Would you be able to elaborate on the alternate peak calling method in section 10.3 of the User Manual?

Does it simply find enriched tiles or do individual tiles get merged into peak features? If you could point me to what happens under the hood in the source code, that would be greatly appreciated. 

I am curious about this alternative peak calling algorithm and wonder if a sliding window approach like csaw could be implemented in ArchR (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4797262/). The main caveat with MACS2 for ATAC peak calling is that it does not account for multi-modal fragment size distributions. MACS2 usually assumes one fragment size, which is not the case in scATAC-seq/ATAC-seq. 

Thanks!

**Comment by rcorces** [comment]
@jgranja24 can provide more explicit details on how the TileMatrix-based peak calling works.
The relevant code block starts here - https://github.com/GreenleafLab/ArchR/blob/ddcaae4a6093685875052219141e5ea41030fc55/R/ReproduciblePeakSet.R#L263

I do think that there are other peak callers (like Genrich for example) out there that could be useful to implement though I'm not sure how high priority this will be. But I'll add the enhancement label anyways.
